src/config/constants.ts

```diff
@@ -165,3 +165,6 @@ export const SETTINGS_OPEN_TIME_LABEL = 'Applications open at:';
 export const SETTINGS_CLOSE_TIME_LABEL = 'Applications close at:';
 export const SETTINGS_CONFIRM_TIME_LABEL = 'Hacker confirmations close at:';
 export const SETTINGS_IS_REMOTE_LABEL = 'Remote hackathon mode';
+
+// Sentry
+export const SENTRY_DSN = 'https://26f1d8550cc34833b9e04894a8192611@o659380.ingest.sentry.io/5764002'
```

Comment:
IMO this should be in an env file and not in version controlled code but that might be out of scope for this PR since generally, most of the things in constants.ts shouldn't be in code (ideally in database or in .env files instead). 

IMO: fine to merge with this here, but a new issue should be created for a more durable well-architected solution for these things.

Reply:
Sounds good, I'll add this to our backlog for a future improvement